wiki/FAQ/en.md

```diff
@@ -274,71 +358,126 @@ _Detailed explanation of scoring can be found at the game style's own respective
 
 ## Performance Points
-*Full explanation: [Performance Points][Performance_Points wikilink]*
+_Full explanation: [Performance Points][Performance Points wikilink]_
 
-Performance Points (pp) is a ranking metric aiming to be more contextually relevant to a continuous game like osu!. It aims to shift the focus from being on the amount of time played to the actual skill of the player. Performance points are heavily based on map difficulty which is determined by a unique algorithm in every gamemode. Based on this difficulty alone each of your scores is rated and assigned a pp value. Your performance rank is predominantly based on your performance on individual maps. The easiest way to improve it is to improve your scores on difficult songs. Do note that game modes are counted separately \[which means, playing (osu!/osu!taiko/osu!catch/osu!mania) raises (osu!/osu!taiko/osu!catch/osu!mania) pp\].
+Performance points (pp) is a ranking metric aiming to be more contextually relevant to a continuous game like osu!. It aims to shift the focus from being on the amount of time played to the actual skill of the player. Performance points is heavily based on map difficulty, determined by a unique algorithm in every game mode. Based on this difficulty alone, each of your scores will be rated and assigned a pp value. Your performance rank is predominantly based on your performance on individual maps. The easiest way to improve it is to improve your scores on difficult songs, and playing more songs. Do note that game modes are counted separately; playing osu!, osu!taiko, osu!catch, or osu!mania will raise the pp for that game mode.
 
-If you have any questions regarding pp, please refer to the [wiki page for pp first][Performance_Points#FAQ wikilink].
+If you have any questions regarding pp, please refer to the [wiki page for pp first][Performance Points#FAQ wikilink].
 
 ---
 
 ## Common issues and questions
 
 ### General
-
 #### Where can I find and get the beatmaps?
 _Detailed explanation of the beatmaps: [Beatmaps][Beatmaps wikilink]_
 
-You can find it from the main page under the name "[Beatmaps][Beatmaplist link]".
--   Just remember to log in to your osu! account before-hand or the server treats you as a guest and prohibit you from downloading beatmaps from the site.
+You can find it from the main page under the name "[Beatmaps][Beatmaplist link]". Just remember to log in to your osu! account beforehand or the server treats you as a guest and prohibit you from downloading beatmaps from the site.
 
-#### Can I disable "Kiai Time"?
-No, because unlike video, they are integrated to the game (as a part with the hit objects), thus cannot be disabled.
+#### Can I disable Kiai Time?
+**No**, because unlike video, it is integrated to the game (as a part with the hit objects), thus cannot be disabled.
 
 #### The maps are too hard! Where can I find some easier maps?
```

Comment:
Adding in `a user named` makes it sounds like the `accountname` person is an alien from outer space (or a bot) in my opinion. I prefer stating the `accountname` as is.

I am leaning to not link the `accountname` directly.